code/src/utest/plugin/lsttokens/ModifyLstTest.java

```diff
@@ -202,6 +202,14 @@ public void testRoundRobinPriority() throws PersistenceLayerException
 		runRoundRobin("MyVar|ADD|3|PRIORITY=1090");
 	}
 
+	//TODO Ignore for now; reactivate later
+//	@Test
```

Comment:
```suggestion
	@Test(enabled=false)
```

Reply:
This doesn't work for me and causes a compile error saying enabled is not a defined annotation for Test.  Is that a JUnit 5 thing?

Reply:
I guess I confused with TestNG. Try @Disabled("Explicitly Disabled") instead.

Change the comment as needed.